### PR TITLE
add instructions for using GitHub to edit docs

### PR DESCRIPTION
**Pull request by agladstein**
closes https://github.com/tskit-dev/msprime/issues/1442

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -105,9 +105,18 @@ is split into two main sections:
...
 
-To get started writing documentation for msprime, first follow the
-{ref}`Quickstart <sec_development_quickstart>`. Once you have created and
-checked out a "topic branch", you are ready to start editing the documentation.
+Documentation can be written on GitHub or locally. If you are new to contributing 
+to msprime and you will be making minor edits to markdown text, you may find
+it easier to make edits on GitHub. To do this, hover your mouse over the GitHub 
+icon in the top right corner of the documentation, and click "suggest edit". 
+You can then edit and preview markdown in GitHub's user interface. Clicking 
+"propose change" at the bottom of the page will commit to a new branch on your fork.
```
Comment (petrelharp) [inline docs/development.md:113]: What happens if you don't already have a fork? (if it doesn't make one, then perhaps insert
"
(you will need to fork the repository if you haven't already).
"

**Review by petrelharp — APPROVED** [review]

File: docs/development.md (hunk 1 above)
Comment (agladstein) [reply docs/development.md:113]: I was wondering that myself. I'll see if I can find someone to double check who doesn't have a fork.

Comment (agladstein) [reply docs/development.md:113]: apparently, it prompts you to make a fork. I'll add that in.

**Review by jeromekelleher — APPROVED** [review]
Looks great, thanks!

**Pull request merged**